docs/data/joy/customization/theme-components/theme-components.md

```diff
@@ -0,0 +1,145 @@
+# Theme components
+
+<p class="description">Learn how to apply custom style to the components at the theme level.</p>
+
+<!-- This page should answer: -->
+<!-- How to customize a component -->
+<!-- How to target specific prop -->
+
+## Component identifier
+
+To theme a specific component, specify the component identifier (`Joy{ComponentImportName}`) inside the `components` node.
```

Comment:
We could start this section with something like: "If you've used Material UI, you are probably familiar with this", so that we can make the correlation to the Material UI's users.